pyproject.toml

```diff
@@ -71,12 +66,15 @@ pymdown-extensions = "*"
 mkdocs-macros-plugin = "*"
 mkdocs-material-extensions = "*"
 markdown-exec = "*"
+
 ###########################################################################
 #                         LINTING CONFIGURATION 
 ###########################################################################
 
+
 [tool.ruff]
 line-length = 150
+fix = true
```

Comment:
It may tends to undefined begaviour in some cases. Sometimes it is impossible to fix linter and you just add ignoring of the rules for a line. With this option, ruff integrated into IDE may break the code when auto-formatting is applied.

Reply:
I don't think that is true; by default `ruff` will only apply safe fixes; they should not be able to break your code. See also [here](https://docs.astral.sh/ruff/linter/#fix-safety). Only if you run with the `--unsafe-fixes` flag should it be able to break your code.